linchpin/linchpin.constants

```diff
@@ -11,10 +11,12 @@ rundb_conn_type = file
 rundb_schema = {"action": "",
                 "inputs": [],
                 "outputs": [],
+                "topology_outputs": [],
                 "cfgs": [],
                 "start": "",
                 "end": "",
                 "rc": 0,
+                "inventory_path": "",
```

Comment:
@herlo : 
inventory_path is generated with respect to per target run . I felt that should be at a higher level since it is independent (and can change frequently for future case where user would like to rerun and update records).